File: decompiled/3.18.0/be/ehealth/businessconnector/mycarenet/memberdata/session/MemberDataService.java
package be.ehealth.businessconnector.mycarenet.memberdata.session;

import be.ehealth.technicalconnector.exception.TechnicalConnectorException;
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationRequest;
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationResponse;

public interface MemberDataService {
   MemberDataConsultationResponse consultMemberData(MemberDataConsultationRequest var1) throws TechnicalConnectorException;
}

File: src/main/kotlin/org/taktik/connector/business/memberdata/service/MemberDataService.kt
package org.taktik.connector.business.memberdata.service

import org.taktik.connector.technical.exception.TechnicalConnectorException
import org.taktik.connector.technical.service.sts.security.SAMLToken
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationRequest
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationResponse

interface MemberDataService {
    @Throws(TechnicalConnectorException::class)
    fun consultMemberData(token: SAMLToken, request: MemberDataConsultationRequest): MemberDataConsultationResponse
}
